doc/design/doi-generation-2.md

```diff
@@ -0,0 +1,212 @@
+# DOI for Draft Dandisets
+
+Authors: Yaroslav O. Halchenko, Dorota Jarecka, Austin Macdonald
+
+## Overview
+
+This document describes an updated strategy for DOI management within the Dandi Archive.
+Upon creation, every public Dandiset will receive a **Dandiset DOI** that will represent the current draft and all future versions.
+Every public published version of a Dandiset will recieve a **Version DOI**.
```

Comment:
```suggestion
Every public published version of a Dandiset will receive a **Version DOI**.
```

Reply:
oh no -- you found a typo which codespell does not know!  

Starting to collect ideas for bot commands etc, codename `@myuniquebot` , so I could 

```
@myuniquebot please submit a PR with a new typo fix for codespell: "recieve" should be "receive"
```

with some shortcut like

```
@myuniquebot new typo: recieve -> receive
```

and it would prepare a PR on my behalf (first in my personal fork to ensure tests pass etc, and then preparing PR to be approved/submitted "as by me")

Reply:
No codespell alerted me to this :)